lib/models/ExistingStatment.dart

```diff
@@ -0,0 +1,62 @@
+// To parse this JSON data, do
+//
+//     final statement = statementFromJson(jsonString);
+
+import 'dart:convert';
+
+List<Statement> statementFromJson(String str) =>
+    List<Statement>.from(json.decode(str).map((x) => Statement.fromJson(x)));
+
+String statementToJson(List<Statement> data) =>
+    json.encode(List<dynamic>.from(data.map((x) => x.toJson())));
+
+class Statement {
```

Comment:
this model is missing many many different things. Please refer to this: https://www.periculum.io/documentation/insights/#responseanalytics

Reply:
I would also rename this file to just Statement.